Add PDS4 language to front page, move Uranus occs box lower, fix typo

matthewtiscareno opened the pull request.

matthewtiscareno commented:
Here is a note from our discussion back on 7/19:
Include job title, then colon and new line, and then name?  Or just name?  Try it both ways and see how it looks.

dstopp commented on website/_data/nav.yaml:
```diff
@@ -87,22 +87,50 @@
       open: new_tab
 
-    - title: Mark Showalter
+    - title: Associate Deputy Manager: Mia Mace
```
after offline discussion, decision was to remove this section of the nav, add a new single nav item that will provide a link to a new page for an 'about us' type of page.

matthewtiscareno commented:
Thanks, @dstopp.  I've removed all changes to nav.yaml, and I've removed mention of the sidebar from the pull request's name. 

If you're good with the remaining changes (all to index.html), then I'd say you might go ahead and merge the branch and deploy to staging.

dstopp commented on website/index.html:
```diff
@@ -46,6 +34,14 @@
 
       ***
 
+      **Browse** our PDS4 data holdings [by clicking here](/pds4/bundles).
```
external to website refs are generally opened in a new tab, so followed by {:target="_blank"}
ie [by clicking here](/pds4/bundles){:target="_blank"}

dstopp commented on website/index.html:
```diff
@@ -67,11 +63,23 @@
       + See the view from Juno at any point in time during the mission, past or future, with the [Jupiter Viewer](/tools/viewer3_jupj.shtml).
 
-      + Create tables of data for Jupiter and its moons with the [Ephemeris Generator](/tools/ephem2_jupj.shtml).
+      + Create tables of data for Jupiter and its moons with the [Ephemeris Generator](/tools/ephem3_jupj.shtml).
```
external to website refs are generally opened in a new tab, so followed by {:target="_blank"}

dstopp reviewed (changes requested):
Note that the actual view is a bit odd w/this order; we may want to consider a different order of boxes just to make the left column not so long in comparison to the right column.
